### PR TITLE
Add playbook for the accounting application.

### PR DESCRIPTION
**Pull request by UmanShahzad**
Adds the playbook that'll run the necessary roles for deploying the [Accounting Automation](https://github.com/open-craft/accounting) service.

See https://github.com/open-craft/ansible-secrets-accounting/pull/1 for actual test instructions.

# BEFORE MERGE

**Make sure to change to use the `master` branch for the role**.

### CODE REVIEW

File: requirements.yml [hunk 1]
```diff
@@ -72,6 +77,10 @@
   src: https://github.com/open-craft/ansible-link-shortener
   version: origin/master
 
+- name: accounting
+  src: https://github.com/open-craft/ansible-accounting
+  version: origin/uman/initial
```
[the remark this answers is not in the document]
Comment (UmanShahzad) [reply requirements.yml]: :+1:

**Comment by UmanShahzad** [comment]
@itsjeyd Updated to use `master` :)

File: requirements.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (itsjeyd) [reply requirements.yml]: @UmanShahzad Ah, I didn't mean to update this *now* -- sorry if that wasn't clear! For testing purposes it makes sense to keep pointing to `uman/initial` here. That way we won't have to tinker with the code locally before running the playbook, nor merge https://github.com/open-craft/ansible-accounting/pull/1 before stepping through the test instructions.

[the remark this answers is not in the document]
Comment (UmanShahzad) [reply requirements.yml]: Alrighty, yeah. I will move it back now for now.

**Comment by UmanShahzad** [comment]
@itsjeyd BTW, I had to delete add a4450d1d9c398a7b18099060550f03150b87e87e because I was getting this message:

```
The following error was encountered:
[Errno 13] Permission denied: '/var/log/letsencrypt/.certbot.lock'
Either run as root, or set --config-dir, --work-dir, and --logs-dir to writeable paths.
```

**Comment by itsjeyd** [comment]
@UmanShahzad OK, thanks for the fix :)

**Pull request merged**